cypress/support/page/common/dashboard.js

```diff
@@ -0,0 +1,34 @@
+class Dashboard {
+  welcomeMessage = '[data-testid="welcome-section"]';
+  dashboardTasksTitle = '[data-testid="dashboard-tasks-title"]';
+  dashboardTaskCourseName = '[data-testid="task-course-name"]';
```

Comment:
I think that we have to make the selector _'private'_. I don't think we are using selectors outside the class. If we need it we have to go via function/method. My stance is we have to make them static and private